createContainer.js

```diff
@@ -41,9 +41,9 @@ module.exports = function createContainer(Component) {
 	return React.createClass({
 		displayName: componentName + "Container",
 		statics: {
-			chargeStores: function(stores, params, callback) {
+			chargeStores: function(stores, params, query, callback) {
```

Comment:
@sokra Sure thing. Which method would you endorse?
- have `query` be the fourth parameter
- do some type checking on passed parameters
- add a condition like `if arguments.length === 4 ...`